src/app/resources/resources.component.html

```diff
@@ -75,16 +75,17 @@ <h3 class="header"><a *ngIf="!nationName" [routerLink]="['/resources/view', elem
           <span class="stars">
             <mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon>
             <!-- used simple arithmetic to get 20 and converted into % -->
-            <span [ngStyle]="{'width': getRating(element.sum, element.timesRated)}">
+            <span [ngStyle]="{'width': getRating(element.has_rated, 1)}">
               <mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon>
             </span>
           </span>
-          <div class="stats stats-total">{{ element.timesRated }} Total</div>
-          <div class="stats stats-female">{{ fRating }}% Female</div>
-          <div class="stats stats-male">{{ mRating }}% Male</div>
+          <div class="stats stats-total">{{ element.total_rating }} Total</div>      
+          <div class="stats"><mat-icon>person-outline</mat-icon>&nbsp;&nbsp;<mat-icon  style="float:right;">person</mat-icon>
+            <mat-progress-bar mode="determinate" [value]="getRatio(element.male_rating, element.female_rating)" [ngClass]="{genderRatingProgressBar:element.total_rating}"></mat-progress-bar>
+          </div>
           <div class="rating">
-            <span class="rating-number">{{ element.timesRated > 0 ? (element.sum / element.timesRated).toFixed(1) : element.sum }}</span>
-            <span>RATING</span>
+            <span class="rating-number">{{ element.total_rating > 0 ? (element.rating / element.total_rating).toFixed(1) : element.rating }}</span>
+            <span>RATE THIS</span>
```

Comment:
Need to add traslation.
Eg; <span i18n>RATE THIS</span>

Reply:
Added Translations, Please review
@nirojdyola 
Thank you!